pages/jobs/[slug].tsx

```diff
@@ -19,7 +19,13 @@ const JobPosting: NextPage<{ job: JobPost }> = ({ job }) => {
 export default JobPosting;
 
 export async function getStaticProps({ params }: { params: { slug: string } }) {
-    const job = (await getJobPost(params.slug)) ?? [];
+    const job = await getJobPost(params.slug);
+
+    if (job === undefined)
+        return {
+            notFound: true,
+        };
+
     return {
         props: { job },
         revalidate: 60 * 60, //After one hour, the cache expires and the page gets rebuilt.
```

Comment:
**Nickpick** : I would be nice to keep the spacing consistent accross comments, i.e."// After" instead of "//After"